libr/core/rvc.c

```diff
@@ -654,6 +654,51 @@ static RList *blobs_add(const char *rp, const RList *files) {
 	return NULL;
 }
 
+R_API char *r_vc_find_rp(const char *path) {
+	{
+		int ret = repo_exists (path);
+		switch (ret) {
+		case 0:
+			break;
+		case 1:
+			return r_str_new (path);
+		case -1:
+			printf ("A corrupted repo may have been found at %s, please refrain from naming any files .rvc\n", path);
+		}
+	}
+	const char *p = r_str_rchr (path, path + strlen (path), *R_SYS_DIR);
+	if (!p) {
+		return NULL;
+	}
+	char *i = malloc (p - path + 1);
+	if (!i) {
+		return NULL;
+	}
+	strncpy (i, path, p - path);
+	printf ("i without %s\n", i); //dbgtrm
```

Comment:
Eprintf or rm completely. Apis shouldnt print stuff to console

Reply:
Yes, dbgtrm stands for debug to remove I forgot to run g/\/\/dbgtrm/norm!dd